src/store/poll/utilities.js

```diff
@@ -0,0 +1,52 @@
+import { database } from '../../../firebase'
+
+const generateRandomNum = () =>
+  Math.floor(Math.random() * 4) + 1
+
+export const generateHash = () =>
+  parseInt([1, 1, 1, 1]
```

Comment:
The hash/key/code thing should probably be a string rather than a number. It's easy to manipulate because we only care about the digits and not, for example, one-thousand-one-hundred-and-eleven

Reply:
Ah shoot, I think I had a reason for a number rather than a string, it might've had something to do with firebase, but I can't think of it. I agree that string is better, I'm just trying to figure out why I did number 🤷‍♂️